tools/src/test/resources/com/nedap/archie/flattener/specexamples/openEHR-EHR-OBSERVATION.empty_observation.v1.0.0.adls

```diff
@@ -9,6 +9,7 @@ description
 
 definition
     OBSERVATION[id1] ∈ {                -- observation
+        uid                             -- existence not changed from reference model
```

Comment:
This class is an example literally from the specification, see https://specifications.openehr.org/releases/AM/latest/ADL2.html#_existence_redefinition_mandation_and_exclusion . I think it should remain the same, and thus not be changed in this PR. The same applies to the archetypes further in this PR. A single archetype containing an UID is enough to prove this works, by the way, since only the parser is the problem. This can just be basic.adl (which is not very basic anymore), or perhaps better, a very small archetype in com.nedap.archie.adlparser, since you are testing the parser here.

Reply:
I always opt for the better option! openEHR-EHR-COMPOSITION.metadata_locatable_uid.v1.0.0.adls it is! 😎